tools/cpp/BUILD.windows.tpl

```diff
@@ -49,11 +49,13 @@ cc_toolchain_suite(
     toolchains = {
         "armeabi-v7a|compiler": ":cc-compiler-armeabi-v7a",
         "x64_windows|msvc-cl": ":cc-compiler-x64_windows",
+        "x64_x86_windows|msvc-cl": ":cc-compiler-x64_x86_windows",
```

Comment:
Should this be `x86_64_windows`?

Reply:
I think we intend to name the host arch as the first and the target arch as second, which is consistent with the vcvarsall.bat's arch argument.
https://sites.google.com/site/windowsdocumentation/visual-stud/visual-studio-communi/manual-pages/vcvarsall